classes/local/breakoutexporter.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+
+namespace mod_zoom\local;
+use csv_export_writer;
+
+class breakoutexporter
+{
+    private $zoom;
+    private $course;
+    private $cm;
+    private $grouping;
+    private $groups;
+    private $groupmembers;
+    /**
+     * @var list of users that we've already seen, and the groups they're in.
+     */
+    private $seenbefore;
+
+    /**
+     * @var bool|int False if the data has never been loaded or a timestamp for when it was updated last.
+     */
+    private $dataloaded = false;
+
+    public function __construct(StdClass $zoom)
```

Comment:
I believe the `s` is lowercase.
```suggestion
    public function __construct(stdClass $zoom)
```